test/extended/util/client.go

```diff
@@ -931,7 +932,15 @@ func (c *CLI) start(stdOutBuff, stdErrBuff *bytes.Buffer) (*exec.Cmd, error) {
 	}
 	cmd := exec.Command(c.execPath, c.finalArgs...)
 	cmd.Stdin = c.stdin
-	framework.Logf("Running '%s %s'", c.execPath, strings.Join(c.finalArgs, " "))
+	// The below log line prints the bearer token in the logs hence redacting
+	// the log if it contains bearer token
+	args := strings.Join(c.finalArgs, " ")
+	if strings.Contains(args, "Bearer") {
+		// replace bearer token with XXXXX string
+		re := regexp.MustCompile(`Authorization:\s+Bearer.*\s+`)
+		args = re.ReplaceAllString(args, "XXXXXXXXXXXXXX ")
```

Comment:
nit: you could also make it clear what you are redacting: 
```suggestion
		args = re.ReplaceAllString(args, "Authorization: Bearer <redacted>")
```

Reply:
Addressed